src/data/search.tsx

```diff
@@ -142,8 +142,7 @@ export function findTopicsByProvisionId(id: string) {
   return result;
 }
 
-// TODO: language for search?
-const constitution = getExpression('eng');
+const constitution = getExpression(localStorage.getItem('locale') || 'en');
```

Comment:
move this into searchLunr.

1. searchLunr must be passed a language, which is the current UI language
2. searchLurnr must check if its search data is in that language; if nothing has been indexed, or if it's the wrong language, re-index
3. perform the search

Reply:
Done. Please me know if I did this correctly, or if I have missed out on a step